src/components/blog/blog-page.js

```diff
@@ -5,7 +5,6 @@ import BlogDetail from './blog-post-card'
 import { urlApiBlog } from '../../urls'
 import { MEDIUM_URL } from '../../consts'
 
-import common from '../../css/page-common-styles.css'
```

Comment:
Why did you remove this CSS import?

Reply:
it was not being anywhere in the file

Reply:
now i understand... forgot that common was in the string. will add it now